### DISCUSSION TITLE
`自定义顶栏`动态、稍后再看、收藏等视频时长（和发布时间）显示效果相关疑问

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#5021] [提问 / Q&A] [answered]
#1866 实装后注意到`自定义顶栏`动态、稍后再看、收藏中的视频卡片的视频时长（动态、收藏的视频发布时间同理）需鼠标指针悬停对应视频卡片才会出现，回看 #4346 截图可知之前至少在我截图的浏览器中是不需要鼠标指针悬停的；
然后我印象中之前好像通过控制台还装过一个可以让`自定义顶栏`的动态中的视频卡片一直显示视频时长还是发布时间的样式（现在找不到那个讨论串了，有可能也不是这个作用，可能记错了，毕竟 #2391 也有一个加入「是否总是显示视频发布时间」选项的需求），加之我目前无法使用之前一直在用的那份配置进行测试，无法得知 #4346 中截图效果是否为上述通过控制台装的样式所致。

所以想问一问 #1866 实装后`自定义顶栏`动态、稍后再看、收藏等视频时长（和发布时间）需鼠标指针悬停对应视频卡片才会出现是刻意为之的吗？

**Comment by the1812** [comment, marked as answer]
控制台是 v1 的操作, v2 有一个正式的组件叫 `总是显示视频时长`

**Reply by Geeyun-JY3** [reply]
还真是，忘记先搜搜有没相关组件了。
